package.json

```diff
@@ -34,5 +34,11 @@
     "jest-diff": "^26.6.2",
     "klaw": "^3.0.0",
     "minimist": "^1.2.5"
+  },
+  "devDependencies": {
+    "ospec": "^4.1.1"
```

Comment:
FYI, `ospec` is a small and fast testing framework we're already using for a bunch of tests ([example](https://github.com/linz/stac/blob/af38f4776db0c5c34fcdf15cffd6f25c4847f217/extensions/linz/tests/linz_collection.test.js)) at Toitū Te Whenua Land Information New Zealand.